packages/web/src/services/solana/WithdrawUSDC.ts

```diff
@@ -24,9 +25,8 @@ const USDC_SLIPPAGE = 3
 export const getFundDestinationTokenAccountFees = async (
   account: PublicKey
 ) => {
-  // TODO: might have to pay rent for root sol account, see BuyAudio.ts
   const rent = await getAssociatedTokenAccountRent()
-  const fee = await getTransferTransactionFee(account)
+  const fee = (await getTransferTransactionFee(account)) ?? 0
```

Comment:
Is this null/undefined because we don't need a transfer, or is it because sometimes we don't get a fee back? If the latter, I think we should have the default be `0.00001` or whatever we're seeing right now as the min transaction fee

Reply:
this returns null sometimes. could you remind where you're getting the min transaction fee from again? this fn returns `5000` when it succeeds which seems like a reasonable value?